packages/sdk/src/assets/collection-legacy.ts

```diff
@@ -9,114 +9,108 @@ import {decodeBridgeableContractDeployedEvent} from '../utils/decoders'
 import {allEventsEnabled, destructSignature, generateRandomSalt, parseBytes} from '../utils/helpers'
 import {evm2hlg, remove0x} from '../utils/transformers'
 import {Erc721Config, GasFee, HolographConfig, Signature, SignDeploy, WriteContractOptions} from '../utils/types'
+import {create2AddressFromDeploymentHash, getErc721DeploymentConfigHash} from '../utils/encoders'
 
 export class HolographLegacyCollection {
-  collectionInfo: CollectionInfo
-  holographConfig: HolographConfig
```

Comment:
Please add this line back, this is really important when we deal with the NFT classes